lib/ads-adal-library/package.json

```diff
@@ -25,5 +25,8 @@
   "dependencies": {
     "axios": "^0.27.2",
     "qs": "^6.9.4"
+  },
+  "resolutions": {
+    "eslint/minimatch": "^3.0.5"
```

Comment:
Did you try updating eslint directly? There's a ton of newer versions which may fix this - and something like eslint is generally going to be straightforward to update.